CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # w3c/vc-di-ecdsa-test-suite  ChangeLog
 
+## 2.2.0 - 2024-01-03
+
+### Changed
+- Get issuer and vc holder used in test data generation from test
+  config. This will allow testers to specify the issuer and/or vc holder for
```

Comment:
```suggestion
- Get issuer and VC holder used in test data generation from test
  config. This will allow testers to specify the issuer and/or VC holder for
```

Reply:
Addressed here [f2e75a2](https://github.com/w3c/vc-di-ecdsa-test-suite/pull/24/commits/f2e75a211c62fe33cfdb9959c492b5ffcf6dd85e)